rest-api-spec/src/main/resources/rest-api-spec/api/sql.get.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
It should be clear that this is different from `sql.query.json`, but don't know if clear enough that all these are `async` SQL APIs. Up to you, not sure if there's a conversion about this.

Reply:
Thanks @bpintea. I used the same naming scheme we have for EQL:

* [eql.delete.json](https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/eql.delete.json)
* [eql.get_status.json](https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/eql.get_status.json)
* [eql.get.json](https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/eql.get.json)

However, I'll add `async` to the SQL versions so its a bit more clear.